File: WebApp/Scripts/app/model/BC_BOX.js
Ext.define('WEBAPP.model.BC_BOX', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'BOXNO', type: 'string' },
        { name: 'BARCODE', type: 'string' },
        { name: 'XCATEGORY', type: 'string' },
        { name: 'DESCRIPT', type: 'string' },
        { name: 'STATUS', type: 'string' },
        { name: 'CREATE_DATE', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_DATE', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
    ]
});

File: WebApp/Models/BC/BC_BOX.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class BC_BOX : JCLib.Mvc.BaseModel
    {
        public string BOXNO { get; set; }
        public string BARCODE { get; set; }
        public string XCATEGORY { get; set; }
        public string DESCRIPT { get; set; }
        public string STATUS { get; set; }
        public string CREATE_USER { get; set; }
        public string CREATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_IP { get; set; }
    }

}